### PR TITLE
fix: fundle support

### LINKED ISSUE #172: Prompt does not work in fundle
## Env

**Pure**: 2.1.5

**Fish**: 3.0.2

**OS/distribution**: macOS 10.14

The prompt used to work in 1.0, but with 2.0, it does not change the prompt. I know that it loads because the functions are available, but there are likely some files that are in the wrong directory.

### PR DESCRIPTION
**Pull request by nesl247**
Fixes Fundle support by moving all functions to functions directory.

Closes #172

### CODE REVIEW

**Comment by edouard-lopez** [comment]
Thanks @nesl247 :+1: 

Can you uncomment the [fundle test case](https://github.com/rafaelrinaldi/pure/blob/8c274175099c82e5a6a8d563ba8b9e887eef1104/tests/installation-methods.test.fish#L70-L82) in tests/installation-methods.test.fish as that's what we want to test.
I don't expect it to work we need it to validate the PR

File: conf.d/pure.fish [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set --universal pure_version 2.1.5 # used for bug report
+set --universal pure_version 2.1.6 # used for bug report
```
Comment (edouard-lopez) [inline conf.d/pure.fish]: check latest release for correctness

Comment (nesl247) [reply conf.d/pure.fish]: You're actually the one that bumped it. I can make changes if necessary.

Comment (edouard-lopez) [reply conf.d/pure.fish]: That's a note for the future, I merge a PR yesterday and release a 2.1.6 version so this need to be updated accordingly.

I haven't had time to finish this PR, sorry.

Comment (nesl247) [reply conf.d/pure.fish]: Oh no problem at all. I thought you were telling me to do it.

**Comment by sgade** [comment]
Hi, I would like to see this PR get merged.
Is there anything else to do, except for merging with master?

I'll try to help out if you need me to.

**Comment by sgade** [comment]
For reference, this seems to be the same change as in https://github.com/rafaelrinaldi/pure/pull/204/ which was merged.

Thank you, @schrodincat!

**Comment by edouard-lopez** [comment]
I'm going to drop `fundle` support, see #232

**Pull request closed without merging**